plugins/checks/functions

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/config/functions"
+
+is_app_checks_enabled() {
+  local desc="return app zero-downtime checks status"
+  local APP="$1"; verify_app_name "$APP"
+  local DOKKU_CHECKS_ENABLED=$(config_get $APP DOKKU_CHECKS_ENABLED)
+
+  if [[ -z "$DOKKU_CHECKS_ENABLED" ]] || [[ "$DOKKU_CHECKS_ENABLED" == "1" ]];then
+    local status=true
+  else
+    local status=false
+  fi
+
+  echo $status
+}
+
+checks_main() {
```

Comment:
Right but this specific function is only used in the command file. In fact, most of the functions in this class aren't used outside the command file.